packages/create/src/actions/create-project.ts

```diff
@@ -35,7 +36,14 @@ export const createProject = async (ctx: Context) => {
 
   spin.start(`Downloading template and creating files...`);
 
-  const defaultTemplate = templates.nextjs;
+  const templatesLookup: Record<Context['framework'], string> = {
+    'Next.js': templates.nextjs,
+    Astro: templates.astro,
+    Remix: '',
```

Comment:
```suggestion
```

Reply:
Removed, and added a type that extracts `Astro` and `Next.js` from the frameworks, to make it all TypeScript happy 👍